docs/guides/nwaku/configure-nwaku.md

```diff
@@ -0,0 +1,246 @@
+
+```bash
+./build/wakunode2 \
+  --websocket-support=true \
+  --websocket-port=[WEBSOCKET LISTENING PORT] \
+  --websocket-secure-support=true \
+  --websocket-secure-key-path=[SECURE WEBSOCKET KEY PATH] \
+  --websocket-secure-cert-path=[SECURE WEBSOCKET CERTIFICATE PATH]
+```
+
+:::info
+Secure WebSocket (`wss`) is not required for local development, as the [secure context restriction](/guides/js-waku/debug-waku-dapp#checking-websocket-setup) does not apply when your website is served locally, like on `localhost` or `127.0.0.1`.
+:::
+
+For example, consider a `nwaku` node that enabled WebSocket for local testing on port `8001`:
+
+```bash
+./build/wakunode2 \
+  --websocket-support=true \
+  --websocket-port=8001
+```
+
+Consider a `nwaku` node that enabled Secure WebSocket using its key and certificate (`privkey.pem` and `fullchain.pem`) on port `8002`:
+
+```bash
+./build/wakunode2 \
+  --websocket-secure-support=true \
+  --websocket-secure-key-path=privkey.pem \
+  --websocket-secure-cert-path=fullchain.pem \
+  --websocket-port=8002
+```
+
+:::tip
+You can use [Let's Encrypt](https://letsencrypt.org/) or [Certbot](https://certbot.eff.org/) to generate a valid certificate for your `nwaku` node:
+
+```bash
+sudo letsencrypt -d <your.domain.name>
+```
+:::
+
+## Configure REST API Server
+
+Nwaku provides a REST API to interact with the node and Waku Network. To enable the REST API, use the following configuration options:
+
+- `rest`: Enables the REST API server on the node (disabled by default).
+- `rest-address` (optional): Listening address of the REST API server. If you omit this option, it will default to `127.0.0.1`.
+- `rest-port` (optional): Listening port of the REST API server. If you omit this option, it will default to `8645`.
+- `rest-relay-cache-capacity`	(optional): Capacity of the Relay REST API message cache.	If you omit this option, it will default to `30`.
+- `rest-admin` (optional): Enables access to REST admin API (disabled by default).
+- `rest-private` (optional): Enables access to REST private API (disabled by default).
+
+```bash
+./build/wakunode2 \
+  --rest=true \
+  --rest-address=[REST SERVER LISTENING ADDRESS] \
+  --rest-port=[REST SERVER LISTENING PORT] \
+  --rest-relay-cache-capacity=[MESSAGE CACHE CAPACITY] \
+  --rest-admin=[true|false] \
+  --rest-private=[true|false]
+```
+
+For example, consider a `nwaku` node that enabled the REST API server on port `9000`:
+
+```bash
+./build/wakunode2 \
+  --rest=true \
+  --rest-port=9000 \
+  --rest-address=127.0.0.1
+```
+
+Consider a `nwaku` node that enabled the REST `admin` and `private` API with a message cache capacity of `100`:
+
+```bash
+./build/wakunode2 \
+  --rest=true \
+  --rest-admin=true \
+  --rest-private=true \
+  --rest-relay-cache-capacity=100
+```
+
+## Configure Filter Protocol
+
+To enable the [Filter protocol](/overview/concepts/protocols#filter) in `nwaku`, use the `filter` configuration option:
```

Comment:
I think best to specify this enables the "server side", ie, "to serve light clients"